cumulusci/tests/test_utils.py

```diff
@@ -337,33 +445,7 @@ def test_zip_clean_metaxml__handles_nonascii(self):
         zf.writestr("classes/test-meta.xml", b"<root>\xc3\xb1</root>")
 
         zf = utils.zip_clean_metaxml(zf)
-        assert b"<root>\xc3\xb1</root>" == zf.read("classes/test-meta.xml")
-
-    def test_doc_task(self):
-        task_config = TaskConfig(
-            {
-                "class_path": "cumulusci.tests.test_utils.FunTestTask",
-                "options": {"color": "black"},
-            }
-        )
-        result = utils.doc_task("command", task_config)
-        assert (
-            """command
-==========================================
-
-**Description:** None
-
-**Class::** cumulusci.tests.test_utils.FunTestTask
-
-extra docs
-
-Options:
-------------------------------------------
-
-* **flavor** *(required)*: What flavor
-* **color**: What color **Default: black**"""
-            == result
-        )
+        assert b"<root>\xc3\xb1</root>" in zf.read("classes/test-meta.xml")
```

Comment:
@Br4nd0R I'm curious why you had to change these to test using `in`? Were there newline differences or something?

Reply:
Switched this back. Not sure why the change was made.